sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -212,12 +212,13 @@ case class Stack(children: Seq[Expression]) extends Generator {
       s"${eval.code}\n$rowData[$row] = ${eval.value};"
     })
 
-    // Create the collection.
+    // Create the collection. Inline to outer class.
```

Comment:
why?

Reply:
Good catch, since we have dependency with `new InternalRow[$numRows]`. We have to inline `new InternalRow[$numRows]`, too.